src/http_upload/mod_http_upload.erl

```diff
@@ -293,3 +280,4 @@ is_positive_integer(_) -> false.
 config_metrics(Host) ->
     OptsToReport = [{backend, s3}], %list of tuples {option, defualt_value}
     mongoose_module_metrics:opts_for_module(Host, ?MODULE, OptsToReport).
+
```

Comment:
Would you mind removing additional newline and correcting `defualt` to `default`? 🙂